public/locales/en/bridge.json

```diff
@@ -4,6 +4,7 @@
   "bridgeModal.heading.collecting": "Bridging Collecting",
   "bridgeModal.heading.success": "Bridging Successful",
   "bridgeModal.heading.disclaimer": "Bridging {{typedValue}} {{symbol}}",
+  "bridge.checkoutBridgeUrl": "For more information, visit <0>{{bridgeName}}↗</0> page",
```

Comment:
Add page inside the anchor element and sounds perfect to me! What do you think?

Reply:
So this is bridgeName + page + icon.

Reply:
Will do!